includes/admin.form.inc

```diff
@@ -48,6 +48,21 @@ function islandora_repository_admin(array $form, array &$form_state) {
             ),
           ),
         ),
+        'sparql_endpoint_externalized' => array(
+          '#type' => 'checkbox',
+          '#title' => t('Use an external SPARQL Query endpoint'),
+          '#description' => t('If an external query endpoint is used, the repository must be configured to update it.'),
+          '#default_value' => variable_get('sparql_endpoint_externalized', FALSE),
+        ),
+        'sparql_endpoint' => array(
+          '#type' => 'textfield',
+          '#title' => t('SPARQL Query Endpoint'),
+          '#default_value' => variable_get('sparql_endpoint', ""),
```

Comment:
Just minor comment: better if called islandora_sparql_endpoint. Also "islandora_".. for the checkbox variable. We don't know if maybe other contributed module (like a sparql one?) could interfere.